src/__tests__/unit.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { lazyForPaint } from '..';
+import { isNodeEnvironment } from '../utils';
+
+jest.mock('../utils', () => ({
+  ...jest.requireActual('../utils'),
+  isNodeEnvironment: jest.fn(),
+}));
+
+describe('lazy', () => {
+  describe('LazyComponent', () => {
+    (isNodeEnvironment as any).mockImplementation(() => true);
+    const mockModuleId = '@foo/bar';
+    const lazyComponent = lazyForPaint(
+      // @ts-ignore - mocking import()
+      () => Promise.resolve({ default: mockModuleId }),
+      {
+        getCacheId: () => '',
+        moduleId: mockModuleId,
+      }
+    );
+    describe('getBundleUrl', () => {
+      it('should return a LazyComponent with a getBundleUrl method on it', () => {
```

Comment:
Is this test necessary? The below test also captures this, and should be enforced by the type system if possible

Reply:
Probably not, will remove